kivy_ios/recipes/hostpython3/allow-cflags-override.patch

```diff
@@ -0,0 +1,13 @@
+--- setuptools.orig/_distutils/sysconfig.py	2023-02-18 18:21:53
++++ setuptools/_distutils/sysconfig.py	2023-02-18 18:23:17
+@@ -329,7 +329,9 @@
+         if 'LDFLAGS' in os.environ:
+             ldshared = ldshared + ' ' + os.environ['LDFLAGS']
+         if 'CFLAGS' in os.environ:
+-            cflags = cflags + ' ' + os.environ['CFLAGS']
++            # cflags = cflags + ' ' + os.environ['CFLAGS']
++            # TODO: we may have found a bug in distutils?
```

Comment:
Is that a TODO for an upstream bug report?

Reply:
Yeah, this is a personal reminder to check blame history and open a bug report on `distutils` upstream if needed 😅

Reply:
(Hopefully will do next weekend)